pom.xml

```diff
@@ -109,6 +109,11 @@
       <artifactId>commons-lang3</artifactId>
       <version>3.6</version>
     </dependency>
+    <dependency>
```

Comment:
Even if this plugin requires Java 1.8, other dependencies require Java 7 which means 2.6.0 for commons-io. For dependency convergence this should be pushed back.

Reply:
Plugins are isolated. Dependency convergence does not matter here. We can drop commons altogether in a new PR imho.

Reply:
@bmarwell What do you mean with "drop commons altogether"? Do you want to remove them from the pom? If so, I wouldn't recommend it because they are used in the plugin and therefore, they should be explicitly named in the pom.

Reply:
On slack there was a discussion that commons could either be shaded or removed altogether for some methods. Currently it is just being used for a few `StringUtils.isNotEmpty( output )` checks. If feasible, the whole dependencies could be replaced by other means.

Reply:
Oh, I see. Thank you for explanation. Then it is fine for me.